src/wallet/wallet.cpp

```diff
@@ -2120,6 +2120,11 @@ bool CWallet::CommitTransaction(CWalletTx& wtxNew, CReserveKey& reservekey)
     return true;
 }
 
+CAmount CWallet::GetRequiredFee(unsigned int nTxBytes)
+{
+    return std::max(minTxFee.GetFee(nTxBytes), ::minRelayTxFee.GetFee(nTxBytes));
```

Comment:
Sad to see another coupling between the Wallet and the Mempool/Node. But it's already all over.
Ideally the wallet should not directly access global mempool variables instead it should get data over a general interface.